### PR TITLE
Solarian Army uniform and armor, plus icon fixes

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Nothing crazy, just adds Sol army stuff for future use and a number of icon fixes.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by KingOfThePing** [comment]
You should document, that you change armour values, too.

File: html/changelogs/dansemacabre-armystrong2.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a Solarian army uniform."
+  - tweak: "Resprited the military modular armor and made it into Solarian military armor."
+  - bugfix: "Fixed a number of icon errors with Solarian uniforms."
```
Comment (KingOfThePing) [inline html/changelogs/dansemacabre-armystrong2.yml:43]:
```suggestion
  - bugfix: "Fixed a number of icon errors with Solarian uniforms."
  - tweak: "Buffs the armour values of the sol armour."
```

**Review by KingOfThePing — CHANGES_REQUESTED** [review]

File: html/changelogs/dansemacabre-armystrong2.yml (hunk 1 above)
Comment (TheDanseMacabre) [reply html/changelogs/dansemacabre-armystrong2.yml:43]: Completely forgot I did that, my bad.

**Review by KingOfThePing — APPROVED** [review]

**Review by KingOfThePing — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**